docs/wiki/guides/current/SoftSerial.md

```diff
@@ -0,0 +1,254 @@
+
+When the GPS is intended for GPS Rescue purposes, Betaflight requires bi-directional communication with the GPS module during the initialisation phase, so that we can configure the data rate, request only the data we need, and block any other junk that we don't need. To configure the GPS requires both Rx and Tx lines. Auto-configuration requires moving through all the available baud rates that the GPS may connect on. For most modern GPS modules, the defdault baud rate is 57600, which is too high for a two-wire Softserial setup to work with. As a result the GPS is likely to not get configured properly, and auto-configuration will fail. Even with a manually-configured GPS, with a single Rx pin on the FC for softserial set to 19200 baud and 2Hz, and with GPS auto-configuration disabled, the best you could hope for would be a very jerky GPS Rescue that may fail at any time.. Hence we strongly discourage the use of SoftSerial for GPS Rescue.
+
+### Softserial with HD OSD (displayport) over MSP
+
+This is not recommended, though it _may_ work. Both a SoftserialTx and an Softserial Rx pin must be defined. The data rate with anything but the simplest OSD would likely overwhelm the connection. Note that softserial is limited to 19200 baud so don't use rates above that.
+
+### Enabling Softserial
+
+Softserial requires custom CLI commands. A unused pin with an associated timer must be found, where the original purpose no longer is needed. The most common are PPM pins or LED_STRIP pins. Not all such pins will work on all boards, and you may need to do some reseach about using softserial on your board before you can make it work.
+
+First go to the CLI and type `resource`, to get a list of the currently assigned pin numbers for each of the currently available pads on the device.
+
+If, for instance, a `LED_STRIP 1` pin is defined to `A15`, and there is a LedStrip pad on the board, and you are not using a led strip, you could de-assign that pin from `LED_STRIP 1` with the CLI command:
+
+`resource led_strip 1 none`
+
+Then the pin can be assigned to the Softserial Port 1 Tx pin with:
+
+`resource serialTx 11 A15`
+
+If the resource reassignment is successful, a `diff` command in CLI will show:
+
+```
+resource LED_STRIP 1 NONE
+resource SERIAL_TX 11 A15
```

Comment:
See above

Reply:
see above

Reply:
You were right; updated the notes to show the new 4.5 form of the command, as wells the old 4.4 and earlier form.